README.md

```diff
@@ -1,13 +1,65 @@
-# Researching HTML elements
+# Leveraging Additional HTML Documentation Resources 
 
-The web is a constantly evolving entity. As we build more ambitious products or learn from our collective needs and mistakes the industry continually evolves the HTML specification by adding new elements or deprecating legacy elements.
+## Problem Statement
 
-We will inevitably come across unfamiliar HTML elements or elements for which we do not fully understand the meaning and/or use. The best place to find these answer is the web itself. The open and free documentation on the web gets better every day.
+As you become familiar with the fundamentals of HTML, you'll get comfortable setting up the
+basics of a web document. However, you will notice that some tags that you'll come across 
+fall outside of your scope of knowledge. 
 
-Let's say we come across an unfamiliar element. We will use the `<samp>` element for our example. We can simply google `samp html element,` and we will find anything we would ever need to know about the element.
+We will inevitably come across unfamiliar HTML elements or elements for which we do not fully 
+understand the meaning and/or use. The best place to find these answer is the web itself. 
+
+## Objectives
+
+1. Use search engines as resource to find information
+2. Observe and identify which resources will contain relevant information
+3. Compare mulitple resources to find the most comprehensive information
+
+
+## Use search engines as resource to find information
+
+We will inevitably come across unfamiliar HTML elements or elements for which we do not fully understand the meaning and/or use. 
+The best place to find these answer is the web itself. The open and free documentation on the web gets better every day.
+
+Let's say in your research, you come across an unfamiliar element like `<samp>`. We can google 
+`samp html element,` and we will find an endless list of results to choose from. Sometimes 
+resources can be outdated, biased, or inaccurate, (like many topics that can be searched 
+online) so it's important to filter through the results to find the best fit. 
 
 ![](https://curriculum-content.s3.amazonaws.com/web-development/html-element-search.png) 
 
-If we select the second search result, we will view the Mozilla Developer Network (**likely the best HTML, JS and CSS documentation source on the internet**) documentation for the `<samp>` element.
+## Observe and identify which resources will contain relevant information
+
+Your instinct may be to check the first result in the search, but bare in mind, that top-
+level search results may not always be the most relevant--they just have higher 
+`Search Engine Optimazation` rankings, which can be impacting by things such as metadata
+and website accessibility, which bump them higher up in the list. If you select the second 
```

Comment:
Suggestion:

> We recommend that you open _the top five_ results and review the results critically. We also recommend that you trust "sites that make the thing" more than others. In the HTML world, the Mozilla Developer Network (MDN) is an extremely trustworthy site. In fact, **it's likely the best HTML, JS and CSS documentation source on the internet**.

As written it seems like we think they should always open the 2nd tab. We want them to be picky and not follow any old site's opinions.